doc/rbd/nvmeof-initiator-linux.rst

```diff
@@ -0,0 +1,83 @@
+==============================
+ NVMe/TCP Initiator for Linux
+==============================
+
+Prerequisites
+=============
+
+- Kernel 4.19 or later
```

Comment:
Where did kernel 4.19 came from?  AFAICT commit https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=3f2304f8c6d6ed97849057bd16fee99e434ca796 landed in kernel 5.0.

Reply:
Accepted. I made this "Kernel 5.10 or later".